cmd/endpoint.go

```diff
@@ -142,6 +147,16 @@ func NewEndpoint(arg string) (ep Endpoint, e error) {
 	}, nil
 }
 
+// checkIPv4 - helper for validating if the provided arg is an ip address.
+func checkIPv4(ipAddress string) bool {
+	host, _, err := net.SplitHostPort(ipAddress)
+	if err != nil {
+		host = ipAddress
+	}
+	// parse the host part of the ipAddress string without port to check if it is an IP without the scheme.
```

Comment:
This comment is unclear to the context.  Either remove the comment or fix it to specific to the context.